### PR TITLE
feat: support pglite

### PR DESCRIPTION
**Pull request by adrienZ**

### CODE REVIEW

File: docs/2.connectors/pglite.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+For this connector, you need to install [`@electric-sql/pglite`](https://www.npmjs.com/package/@electric-sql/pglite) dependency:
+
+:pm-i{name="@electric-sql/pglite"}
+
+Use `pglite` connector:
+
+```js
+import { createDatabase, sql } from "db0";
+import sqlite from "db0/connectors/pglite";
```
Comment (adrienZ) [inline docs/2.connectors/pglite.md:21]: Typo

**Comment by hareland** [comment]
Would be really awesome to have this added 🙏

File: src/connectors/pglite.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { PGlite, type PGliteOptions } from "@electric-sql/pglite";
+import type { Connector, Statement } from "../types";
+
+export type ConnectorOptions = PGliteOptions
```
Comment (pi0) [inline src/connectors/pglite.ts:4]: I have simplified types for now to avoid duplication and possibly going out of sync

**Comment by pi0** [comment]
Thanks ❤️

**Pull request merged**